data/monster/monsters/scarlett_etzel.xml

```diff
@@ -0,0 +1,68 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<monster name="Scarlett Etzel" nameDescription="Scarlett Etzel" race="blood" experience="20000" speed="175">
+	<health now="30000" max="30000" />
+	<look type="1201" corpse="34109" />
+	<targetchange interval="2000" chance="5" />
+	<flags>
+		<flag attackable="1" />
+		<flag hostile="1" />
+		<flag summonable="0" />
+		<flag convinceable="0" />
+		<flag illusionable="0" />
+		<flag isboss="1" />
+		<flag ignorespawnblock="0" />
+		<flag pushable="0" />
+		<flag canpushitems="1" />
+		<flag canpushcreatures="1" />
+		<flag staticattack="90" />
+		<flag targetdistance="1" />
+		<flag canwalkonenergy="1" />
+		<flag canwalkonfire="1" />
+		<flag canwalkonpoison="1" />
+	</flags>
+	<attacks>
+		<attack name="melee" interval="2000" min="0" max="-600" />
+		<attack name="death" interval="2000" chance="16" distance="7" target="1" min="-400" max="-600" />
```

Comment:
need to add shootEffect and areaEffect for the projectile and mort animations. pretty sure "target" is not used
```suggestion
		<attack name="death" interval="2000" chance="16" distance="7" min="-400" max="-600" />
			<attribute key="shootEffect" value="suddendeath" />
			<attribute key="areaEffect" value="mortarea" />
```